src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -61,8 +63,10 @@ public PersonCard(Person person, int displayedIndex) {
         String telegramAddress = person.getTelegramHandle() != null ? " (@" + person.getTelegramHandle().value + ")"
                 : " -";
         name.setText(person.getName().fullName + telegramAddress);
-        attendance.setText(String.format("Attendance: %s",
-                this.person.getStyledAttendanceList("✓", "✕")));
+        attendance.setText(String.format("Attendance: %d/%d  Attendance summary: %s",
```

Comment:
Is this a bit terse? It is obvious from context that the following `%s` is a summary - shouldn't we omit `Attendance summary` in favour of a more minimalistic display?